docs/technical_reference/unit_models/translators/translator_asm1_adm1.rst

```diff
@@ -0,0 +1,194 @@
+
+   "Time", ":math:`t`", "[0]"
+   "Inlet/outlet", ":math:`x`", "['in', 'out']"
+   "Phases", ":math:`p`", "['Liq']"
+   "Inlet Components", ":math:`j`", "['H2O', 'S_I', 'S_S', 'X_I', 'X_S', 'X_BH', 'X_BA', 'X_P', 'S_O', 'S_NO', 'S_NH', 'S_ND', 'X_ND', 'S_ALK']"
+   "Ion", ":math:`j`", "['S_cat', 'S_an'] \  :sup:`*`"
+   "Outlet Components", ":math:`j`", "['H2O', 'S_su', 'S_aa', 'S_fa', 'S_va', 'S_bu', 'S_pro', 'S_ac', 'S_h2', 'S_ch4', 'S_IC', 'S_IN', 'S_I', 'X_c', 'X_ch', 'X_pr', 'X_li', 'X_su', 'X_aa', 'X_fa', 'X_c4', 'X_pro', 'X_ac', 'X_h2', 'X_I', 'S_cat', 'S_an', 'S_co2']"
+
+**Notes**
+ :sup:`*` Ion" is a subset of "Component" and uses the same symbol j.
+
+.. _Translator_ASM1_ADM1_parameters:
+
+Parameters
+----------
+
+.. csv-table::
+   :header: "Description", "Symbol", "Parameter Name", "Value", "Units"
+
+   "Nitrogen fraction in particulate products", ":math:`i_{xe}`", "i_xe", 0.06, ":math:`\text{dimensionless}`"
+   "Nitrogen fraction in biomass", ":math:`i_{xb}`", "i_xb", 0.08, ":math:`\text{dimensionless}`"
+   "Anaerobic degradable fraction of X_I and X_P", ":math:`f_{xI}`", "f_xI", 0.05, ":math:`\text{dimensionless}`"
+
+.. _Translator_ASM1_ADM1_equations:
+
+Equations and Relationships
+---------------------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Pressure balance", ":math:`P_{out} = P_{in}`"
+   "Temperature balance", ":math:`T_{out} = T_{in}`"
+   "Volumetric flow equality", ":math:`F_{out} = F_{in}`"
+   "Total Kjeldahl nitrogen", ":math:`TKN = S_{NH} + S_{ND} + X_{ND} + i_{xb}(X_{BH} + X_{BA}) + i_{xe}(X_{I} + X_{P})`"
+
+COD Equations
+-------------
+The total incoming COD is reduced in a step-wise manner until the COD demand has been satisfied. The reduction is based on a
+hierarchy of ASM1 state variables such that :math:`S_s` is reduced by the COD demand first. If there is insufficient
+Ss present, then :math:`S_s` is reduced to zero and the remaining demand is subtracted from :math:`X_s`. If necessary, :math:`X_BH` and :math:`X_BA` may also need to be reduced.
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "COD demand", ":math:`COD_{demand} = S_{o} + 2.86S_{NO}`"
+   "Readily biodegradable substrate remaining (step 1)", ":math:`S_{S, inter} = S_{S} - COD_{demand}`"
+   "Slowly biodegradable substrate remaining (step 2)", ":math:`X_{S, inter} = X_{S} - COD_{demand, 2}`"
+   "Active heterotrophic biomass remaining (step 3)", ":math:`X_{BH, inter} = X_{BH} - COD_{demand, 3}`"
+   "Active autotrophic biomass remaining (step 4)", ":math:`X_{BA, inter} = X_{BA} - COD_{demand, 4}`"
+   "Soluble COD", ":math:`COD_{s} = S_{I} + S_{S, inter}`"
+   "Particulate COD", ":math:`COD_{p} = X_{I} + X_{S, inter} + X_{BH, inter} + X_{BA, inter} + X_{P}`"
+   "Total COD", ":math:`COD_{t} = COD_{s} + COD_{p}`"
+
+S_nd and S_s Mapping Equations
+------------------------------
+
+.. figure:: ../../../_static/unit_models/translators/mapping_step_a.jpg
```

Comment:
We need to reference any figures taken from the literature. Also, we should include the relevant reference(s) towards the end of the documentation from which the formulation of this translator block (Copp Interface if I'm not mistaken) originated from.

Reply:
Oh, you did the latter!